keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -43,12 +45,22 @@ class ApplicationController(
   ): Map<String, Any> {
     if (includeDetails) {
       val resources = resourceRepository.getSummaryByApplication(application)
+      val hasDeliveryConfig = deliveryConfigRepository.hasDeliveryConfig(application)
+      val constraintStates = if (hasDeliveryConfig) {
+        deliveryConfigRepository.constraintStateFor(application)
+      } else {
+        emptyList()
+      }
 
       return mapOf(
         "hasManagedResources" to resources.isNotEmpty(),
-        "resources" to resources
+        "resources" to resources,
+        "hasDeliveryConfig" to hasDeliveryConfig,
```

Comment:
Just for context, what's the idea behind the `hasXyz` properties in the output here? For `hasManagedResources` at least, it seems to be redundant with `resources`?

Reply:
I think this is really a `deck` question for @erikmunson but let's discuss outside of the context of this PR.